<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>W820NBPlusForm</class>
 <widget class="QWidget" name="W820NBPlusForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>870</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="infoGroup">
     <property name="title">
      <string>Info</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Battery:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="batteryLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="batteryGetButton">
          <property name="text">
           <string>Get</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>MAC Address:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="MACLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="MACGetButton">
          <property name="text">
           <string>Get</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Firmware Version:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="firmwareLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="firmwareGetButton">
          <property name="text">
           <string>Get</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="nameGroup">
     <property name="title">
      <string>Name</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Takes effect after removing the pairing record and re-pairing</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nameSetButton">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="noiseGroup">
     <property name="title">
      <string>Noise Reduction</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="noiseNormalButton">
        <property name="text">
         <string>Normal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="noiseReductionButton">
        <property name="text">
         <string>Noise Reduction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="noiseAmbientSoundButton">
        <property name="text">
         <string>Ambient Sound</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ambientSoundGroup">
     <property name="title">
      <string>Ambient Sound</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QSlider" name="ASSlider">
        <property name="minimum">
         <number>-3</number>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ASBox">
        <property name="minimum">
         <number>-3</number>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ASSetButton">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="controlSettingsGroup">
     <property name="title">
      <string>Control Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QCheckBox" name="CSNormalBox">
          <property name="text">
           <string>Normal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CSNoiseReductionBox">
          <property name="text">
           <string>Noise Reduction</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CSAmbientSoundBox">
          <property name="text">
           <string>Ambient Sound</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Please select at least 2 items</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="soundEffectGroup">
     <property name="title">
      <string>Sound Effect</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QRadioButton" name="SENormalButton">
        <property name="text">
         <string>Normal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="SEPopButton">
        <property name="text">
         <string>Pop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="SEClassicalButton">
        <property name="text">
         <string>Classical</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="SERockButton">
        <property name="text">
         <string>Rock</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="promptVolumeGroup">
     <property name="title">
      <string>Prompt Volume</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QSlider" name="PVSlider">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>7</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="PVBox">
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="value">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PVSetButton">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="shutdownTimerGroup">
     <property name="title">
      <string>Shutdown timer</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QSlider" name="STSlider">
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="pageStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="STBox">
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="STSetButton">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="LDACGroup">
     <property name="title">
      <string>LDAC</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="LDACOFFButton">
          <property name="text">
           <string>OFF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="LDAC48kButton">
          <property name="text">
           <string>44.1k/48k</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="LDAC96kButton">
          <property name="text">
           <string>96k</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Takes effect after re-pairing</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="gameModeBox">
     <property name="text">
      <string>Game Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="poweroffButton">
     <property name="text">
      <string>Poweroff</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="disconenctButton">
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="re_pairButton">
     <property name="text">
      <string>Re-pair</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string>Factory Reset</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
